buildSrc/src/main/resources/pom.xml

```diff
@@ -21,6 +21,11 @@
             <groupId>org.testcontainers</groupId>
             <artifactId>oracle-xe</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <version>4.5.1</version>
```

Comment:
the latest is v5.1.0, is there an issue with using v5.x with Micronaut? If so, this should be described in the guide

Reply:
resolved via https://github.com/micronaut-projects/micronaut-guides/pull/900/commits/fb9f33fea70fa0f24a8abad8a6a695655fbd089a